<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preview_Window</class>
 <widget class="QMainWindow" name="Preview_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>464</width>
         <height>357</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>6</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuCreate_window">
     <property name="title">
      <string>Create window</string>
     </property>
     <addaction name="actionQMainWindow"/>
     <addaction name="actionQDialog"/>
     <addaction name="separator"/>
     <addaction name="actionQDockWidget"/>
    </widget>
    <addaction name="menuCreate_window"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionSome_item_1">
   <property name="text">
    <string>some item 1</string>
   </property>
  </action>
  <action name="actionSome_item_2">
   <property name="text">
    <string>some item 2</string>
   </property>
  </action>
  <action name="actionSome_item_4">
   <property name="text">
    <string>some item 4</string>
   </property>
  </action>
  <action name="actionSome_item_5">
   <property name="text">
    <string>some item 5</string>
   </property>
  </action>
  <action name="actionSome_item_6">
   <property name="text">
    <string>some item 6</string>
   </property>
  </action>
  <action name="actionSome_item_7">
   <property name="text">
    <string>some item 7</string>
   </property>
  </action>
  <action name="actionFirst_item">
   <property name="text">
    <string>first item</string>
   </property>
  </action>
  <action name="actionSecond_item">
   <property name="text">
    <string>second item</string>
   </property>
  </action>
  <action name="actionQMainWindow">
   <property name="text">
    <string>QMainWindow</string>
   </property>
  </action>
  <action name="actionQDialog">
   <property name="text">
    <string>QDialog</string>
   </property>
  </action>
  <action name="actionQDockWidget">
   <property name="text">
    <string>QDockWidget</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
